app/alarm/freetts-annunciator/src/main/java/org/phoebus/applications/alarm/freetts/annunciator/FreeTTSAnnunciator.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Oak Ridge National Laboratory.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.phoebus.applications.alarm.freetts.annunciator;
+
+import com.sun.speech.freetts.Voice;
+import com.sun.speech.freetts.VoiceManager;
+import org.phoebus.applications.alarm.ui.annunciator.Annunciator;
+
+/**
+ * Annunciator class. Uses freeTTS to annunciate passed messages.
+ * @author Evan Smith, Kunal Shroff
+ */
+@SuppressWarnings("nls")
+public class FreeTTSAnnunciator implements Annunciator
+{
+    private final VoiceManager  voiceManager;
+    private final Voice         voice;
+    private static final String voice_name = "kevin16";
```

Comment:
I know @kasemir said only the kevin16 voice worked well but I'm wondering if it is worth making this a runtime preference setting and default it to kevin16 ?

Reply:
As far as I remember, the other voices not only don't work well, they're useless. Really wouldn't bother.

Reply:
Like one is only for numbers, something like that